concrete/controllers/single_page/dashboard/system/express/entities/forms.php

```diff
@@ -316,15 +320,31 @@ public function edit($id = null)
         }
     }
 
+    protected function canDeleteForm(Entity $entity, Form $form)
+    {
+        $canDeleteForm = true;
+        if (
+            ($entity->getDefaultEditForm() && $entity->getDefaultEditForm()->getID() == $form->getID()) ||
+            ($entity->getDefaultViewForm() && $entity->getDefaultViewForm()->getID() == $form->getID())
```

Comment:
Yeah, I've just always thought it sketchy to rely on object equivalence since who knows what might happen in the future.